ts/navigation/MainNavigator.tsx

```diff
@@ -44,16 +45,37 @@ const getIcon = (routeName: string): string => {
   return route === undefined ? fallbackIcon : route;
 };
 
-const styles = StyleSheet.create({
-  tabBarStyle: {
-    height: 64,
+const tabBarStyleFactory = () => {
+  const defaultStyle = {
     backgroundColor: variables.colorWhite,
     paddingLeft: 3,
     paddingRight: 3,
     borderTopWidth: 0,
-    paddingTop: 8,
-    paddingBottom: 10
-  },
+    paddingTop: 8
+  };
+
+  // Add space to the bottom bar for iPhone 12, which has
+  // deviceId "iPhone 13" counterintuitevely
+  // See https://gist.github.com/adamawolf/3048717 for device IDs
+
+  const isIPhone12 = deviceInfoModule.getDeviceId().indexOf("iPhone13") !== -1;
+
+  return {
+    ...defaultStyle,
+    ...(isIPhone12
+      ? {
+          paddingBottom: 20,
+          height: 74
```

Comment:
Trying this solution I just notice some minimal differences between iPhone 11 and 12 space between home button and navigator bar

<img src="https://user-images.githubusercontent.com/3959405/108237087-d7149880-7147-11eb-9ff9-4bc5fe01779d.png" height="550"/>

i just tried out with 
```typescript
        {
          paddingBottom: 30,
          height: 84
        }
```

and seems to be closer as style what do you think?

Reply:
Yes! Applying this change.